### PR TITLE
Footer: added copy for CC license

### PR DESCRIPTION
**Pull request by Yashs911**
Related issue: #1379 
Summary: 
- [x] Removed "License" copy
- [x] Organized items by alphabetical order (except for "Donate", which should remain the first link)
- [x] Added "Most content available under a Creative Commons license." Link for Creative Commons license is here
- [x] Changed Mozilla logo to the most updated one. Logo here
- [x] Change copy to be consistent with other web properties: "Join our Mozilla News email list to take action and stay updated!"

If you don't know who to ask for a review, try @pomax , @cadecairos , or @mmmavis. If you do, please delete this line.

### CODE REVIEW

**Comment by mmmavis** [comment]
Thanks @Yashs911 ! I will review this once you have access to the new logo & have this PR updated!

**Comment by Yashs911** [comment]
Hi, @mmmavis PR updated

File: components/footer/footer.jsx [hunk 1]
```diff
@@ -93,8 +88,9 @@ export default class Footer extends React.Component {
               <div className="my-5 mb-md-0">
                 <a className="logo" href="https://foundation.mozilla.org">
                   <img
-                    src="../assets/svg/mozilla-block-white.svg"
+                    src="../assets/svg/mozilla-block-white.8a5ea59a8186.svg"
```
Comment (mmmavis) [inline components/footer/footer.jsx:91]: please rename the image to `mozilla-block-white.svg` and this line can stay unchanged

File: components/footer/footer.jsx [hunk 2]
```diff
@@ -93,8 +88,9 @@ export default class Footer extends React.Component {
...
                     alt="Mozilla Foundation Home Page"
+                    style={{ width: 96, height: 31 }}
```
Comment (mmmavis) [inline components/footer/footer.jsx:93]:
```suggestion
                    width="96"
```

File: components/footer/footer.jsx [hunk 3]
```diff
@@ -119,6 +115,10 @@ export default class Footer extends React.Component {
                     <a href="https://foundation.mozilla.org">
                       foundation.mozilla.org
                     </a>
+                    .Most content available under a{" "}
```
Comment (mmmavis) [inline components/footer/footer.jsx:118]:
```suggestion
                    . Most content available under a{" "}
```

File: components/newsletter-sign-up/newsletter-sign-up.jsx [hunk 4]
```diff
@@ -272,7 +272,7 @@ class NewsletterSignUp extends React.Component {
 
 NewsletterSignUp.defaultProps = {
   ctaHeader: `Protect the internet as a global public resource`,
-  ctaDescription: `Join our email list to take action and stay updated!`,
+  ctaDescription: `Join our Mozilla News email list to take action and stay updated!`,
```
Comment (mmmavis) [inline components/newsletter-sign-up/newsletter-sign-up.jsx:275]: I would change line 137-141 to 
```jsx
<div
  dangerouslySetInnerHTML={{
    __html: this.props.ctaDescription,
  }}
/>
```
and this line to
```suggestion
  ctaDescription: `<p>Join our <b>Mozilla News</b> email list to take action and stay updated!</p>`,
```

**Review by mmmavis — CHANGES_REQUESTED** [review]
Few requests!

**Comment by Yashs911** [comment]
@mmmavis Have made the changes as requested

**Review by mmmavis — APPROVED** [review]
✨

**Pull request merged**